dev/com.ibm.ws.microprofile.faulttolerance.1.1/bnd.bnd

```diff
@@ -0,0 +1,59 @@
+#*******************************************************************************
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8
+
+Bundle-Name: com.ibm.ws.microprofile.faulttolerance.1.1
```

Comment:
I don't think this bundle is needed at all, it doesn't add anything, it just copies the contents of the 1.0 version.